objects/src/transaction/consumed_notes.rs

```diff
@@ -199,3 +202,47 @@ impl From<&RecordedNote> for ConsumedNoteInfo {
         Self::new(recorded_note.note().nullifier(), recorded_note.note().script().hash())
     }
 }
+
+// SERIALIZATION
+// ================================================================================================
+//
+
+impl Serializable for ConsumedNotes {
+    fn write_into<W: ByteWriter>(&self, target: &mut W) {
+        target.write_u64(self.notes.len() as u64);
+        self.notes.write_into(target);
+        self.commitment.write_into(target);
```

Comment:
Similar to other comments, I'm wondering if we should serialize the commitment or compute it "on demand".

Reply:
changed